controllers/searchoperator_controller.go

```diff
@@ -345,12 +350,36 @@ func (r *SearchOperatorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			}
 		})
 
+	releaseName, ok := os.LookupEnv("RELEASE_NAME")
+	var apiSecretName string
+	if ok {
+		apiSecretName = strings.Trim(releaseName, " ") + "-search-api-secrets"
+	}
```

Comment:
Would it be simpler to add a label to the secret and use the label to find it?

Reply:
Secrets are generated by OCP Service Serving Certificates operator.